src/utils/Fishable.ts

```diff
@@ -18,6 +18,7 @@ export interface Metadata {
   id: string;
   name: string;
   sdType?: string;
+  ancestor?: string;
```

Comment:
I don't love the term `ancestor` for this.  At first I thought `resourceType` would be more accurate, but now I see that is not quite correct in the case of `Instance`s.  But I still feel like `ancestor` is kind of a deceptive name.  For example, someone might (reasonably) think that the `ancestor` of `MyConditionProfile` is `Condition`, not `StructureDefinition`.  Should this just be `instanceOf`?  A profile is an instance of a `StructureDefinition` after all...

Reply:
Ok so https://github.com/FHIR/sushi/pull/870/commits/c39f752629eb57bdc5f31f3fa9e0f72aa7bbdd31 renames `ancestor` to `resourceType`. The one complication of this is that on an `Instance` defined in the `FSHTank`, we don't yet know the `resourceType`. So in this case, the `FSHTank` returns nothing. The `MasterFisher` however knows a bit more, so it can fish for the `InstanceOf` on the `Instance`, and use the `sdType` of the result, and that should be the `resourceType` of the `Instance`. This does clarify the keyword, but it comes at the cost of some additional complexity and additional `fish`ing operations. Let me know if you think that ends up tipping the balance to making it not worth it. In that case, I think we can just call the keyword `instanceOf`, even if that is a bit tough to swallow.